tests/testthat/test-with-abort.txt

```diff
@@ -13,10 +13,9 @@ message: Low-level message
 class:   `rlang_error`
 backtrace:
  1. rlang::with_abort(f())
- 3. rlang:::f()
- 4. rlang:::g()
- 5. rlang:::h()
- 6. base::stop("Low-level message")
+ 4. rlang:::f()
+ 5. rlang:::g()
+ 6. rlang:::h()
```

Comment:
Is it confusing to not include the final call to `stop()`? I could go either way.

Reply:
One advantage is that it is consistent with backtraces created by `abort()`.

Reply:
I think I'd definitely prefer to see the error message in tree call tree

Reply:
OTOH while `traceback()` shows them, neither `where` nor `recover` does